elasticlib/Elastic.java

```diff
@@ -35,13 +35,13 @@ public static void sendAlert(ElasticNotification alert) {
   }
 
   /**
-   * Represents a notification object to be sent to the Elastic dashboard. This object holds
+   * Represents an alert object to be sent to the Elastic dashboard. This object holds
    * properties such as level, title, description, display time, and dimensions to control how the
    * alert is displayed on the dashboard.
    */
-  public static class ElasticNotification {
+  public static class Alert {
```

Comment:
I prefer keeping this as ElasticNotification or Notification to avoid it getting confused with WPILib Alerts, since these serve a different purpose